tests/image/test_init.py

```diff
@@ -15,9 +15,9 @@
 
 @pytest.fixture
 def project_dir(project_dir):
-    """Overriden project_dir fixture without Dockerfile"""
-    dockerfile_path = os.path.join(project_dir, 'Dockerfile')
-    os.remove(dockerfile_path)
+    """Overriden project_dir fixture without Dockerfile & setup.py."""
+    os.remove(os.path.join(project_dir, 'Dockerfile'))
+    os.remove(os.path.join(project_dir, 'setup.py'))
```

Comment:
if you have separate test case for setup.py creation - it makes sense to keep fake setup.py for all other test cases to have both cases covered. just remove it in an appropriate test case.